src/components/coblocks-deactivate-modal/test/modal.cypress.js

```diff
@@ -0,0 +1,57 @@
+describe( 'Component: CoBlocks Deactivate Modal', () => {
+	beforeEach( () => {
+		cy.visit( Cypress.env( 'testURL' ) + '/wp-admin/plugins.php' );
+
+		cy.get( '#deactivate-coblocks' ).should( 'exist' );
+	} );
+
+	afterEach( () => {
+		cy.get( '#activate-coblocks' ).should( 'exist' );
+
+		cy.get( '#activate-coblocks' ).click();
+	} );
+
+	it( 'open modal and submit feedback', function() {
+		cy.intercept(
+			'GET',
+			'https://wpnux.godaddy.com/v3/api/feedback/coblocks-optout?domain=http://localhost:8889&random=1&language=en-US',
+			{ fixture: '../.dev/tests/cypress/fixtures/network/coblocks_optout.json' }
+		);
+
+		cy.wait( 1000 );
```

Comment:
In theory, you don't need to wait, as you just verified in your `beforeEach` that `#deactivate-coblocks` exists.